scss/core/base.scss

```diff
@@ -356,22 +356,25 @@ table.mce-item-table {
   box-shadow: none;
 
   &:not(:hover) {
-    box-shadow: 0px 0px 0px $focusOutlineWidth $focusOutlineColor;
+    box-shadow: 0px 0px 0px $focusOutlineWidth $focusOutlineColor !important;
+    outline: none;
   }
   
   @include above($tabletBreakpoint) {
     box-shadow: none;
 
     &:not(:hover) {
-      box-shadow: 0px 0px 0px $focusOutlineWidthTablet $focusOutlineColorTablet;
+      box-shadow: 0px 0px 0px $focusOutlineWidthTablet $focusOutlineColorTablet !important;
+      outline: none;
```

Comment:
This is the same static value for all breakpoints. Why can't we just have this declared once for mobile, which will them apply to all breakpoints?

```suggestion
```

Reply:
Fixed